docs/tutorials/configure-external-signer-tls.md

```diff
@@ -145,22 +143,15 @@ web3signer --key-store-path=/Users/me/keyFiles/ \
 --tls-keystore-file=/Users/me/certs/web3signer_keystore.p12 \
 --tls-keystore-password-file=/Users/me/certs/web3signer_keystore_password.txt \
 --tls-known-clients-file=/Users/me/certs/knownClients.txt \
-eth2 --slashing-protection-enabled=false
 ```
 
 ## 5. Start Teku
 
 Start Teku and specify the [keystore](#teku-keystore-and-password-file) and [truststore](#2-create-the-truststore-and-password-file) created earlier, with the accompanying password files.
 
```

Comment:
```suggestion

:::note
This example connects to the Holesky testnet, and connects to the execution client's JSON-RPC URL at `http://127.0.0.1:8551`.
:::

```

I think this should suffice for mentioning the configurable options of testnet and RPC URL to connect to. Anyone looking to run Web3Signer should be familiar with the available testnets and won't need additional context on them.

Reply:
In the current doc we linked out to Goerli and Infura - I guess to help users with the sign up and onboarding. Holesky seems top be relatively new (I think announced in Sept 2023). Working on this PR, it had to be checked what testnet to connect to and if there were docs for Holesky. Do you think if we added the Holesky link to the text you provided, that might be helpful for someone using this doc? It might also help with findability.

Reply:
I don't think a link is absolutely necessary, but if you think it is, you can scratch this suggestion and see my new suggestion on line 25.